src/main/java/com/SBS/Models/OddsFactory.java

```diff
@@ -0,0 +1,19 @@
+package com.SBS.Models;
+
+import java.util.List;
+
+public class OddsFactory {
+    private OddsBasketball oddsBasketball;
+    private OddsSoccer oddsSoccer;
+    private OddsTennis oddsTennis;
+    private Discipline discipline;
+    private List<Bet> odds;
+
+    public List<Bet> createOdds(Discipline discipline) {
```

Comment:
coś tu nazewnictwo się nie zgadza. OddsFactory i createOdds sugeruje ze zwrocisz Oddsy, a zwaracasz betsy

Reply:
Oddsy są listą Betów, może powinienem to nazwać jako bets w takim razie. i BetsFactory. W bukmacherce po prostu Odds to są kursy przypisane do meczu a user wybiera jeden i nazywa się to Bet, więc tak to zapisałem, ale mogę ujednolicić jak trzeba